Cython/Compiler/ExprNodes.py

```diff
@@ -7047,6 +7029,32 @@ def analyse_as_type_attribute(self, env):
                     error(self.pos, "%s not a known value of %s" % (self.attribute, type))
         return None
 
+    def _create_ubcm_entry(self, type, entry, env):
+        # Create a temporary entry describing the unbound C method in `entry`
+        # as an ordinary function.
+        if entry.func_cname and entry.type.op_arg_struct is None:
+            cname = entry.func_cname
+            if entry.type.is_static_method or (
+                    env.parent_scope and env.parent_scope.is_cpp_class_scope):
+                ctype = entry.type
+            elif type.is_cpp_class:
+                error(self.pos, "%s not a static member of %s" % (entry.name, type))
+                ctype = PyrexTypes.error_type
+            else:
+                # Fix self type.
+                ctype = copy.copy(entry.type)
+                ctype.args = ctype.args[:]
+                ctype.args[0] = PyrexTypes.CFuncTypeArg('self', type, 'self', None)
```

Comment:
I understand that this code was not changed, but what about keyword arguments? Are they correctly handled as well? I wouldn't be surprised if we didn't have any tests for keyword arguments in C++ functions. (There's `cdef_function_kwargs.pyx` for C functions.)

Reply:
Tried it, looks like it doesn't even work. Let's ignore this issue here.